[2022-09-08T18:49:21.524+0500] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: expectation_dag.Expect_unique manual__2022-09-08T13:49:05.304007+00:00 [queued]>
[2022-09-08T18:49:21.529+0500] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: expectation_dag.Expect_unique manual__2022-09-08T13:49:05.304007+00:00 [queued]>
[2022-09-08T18:49:21.529+0500] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T18:49:21.529+0500] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-08T18:49:21.529+0500] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T18:49:21.543+0500] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): Expect_unique> on 2022-09-08 13:49:05.304007+00:00
[2022-09-08T18:49:21.548+0500] {standard_task_runner.py:52} INFO - Started process 1069683 to run task
[2022-09-08T18:49:21.551+0500] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'expectation_dag', 'Expect_unique', 'manual__2022-09-08T13:49:05.304007+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/expectation_dag.py', '--cfg-path', '/tmp/tmphucwacwb', '--error-file', '/tmp/tmpprh8la3t']
[2022-09-08T18:49:21.552+0500] {standard_task_runner.py:80} INFO - Job 31: Subtask Expect_unique
[2022-09-08T18:49:21.609+0500] {task_command.py:371} INFO - Running <TaskInstance: expectation_dag.Expect_unique manual__2022-09-08T13:49:05.304007+00:00 [running]> on host shahroze
[2022-09-08T18:49:21.659+0500] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=expectation_dag
AIRFLOW_CTX_TASK_ID=Expect_unique
AIRFLOW_CTX_EXECUTION_DATE=2022-09-08T13:49:05.304007+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-08T13:49:05.304007+00:00
[2022-09-08T18:49:21.664+0500] {logging_mixin.py:115} INFO - {
  "meta": {},
  "exception_info": {
    "raised_exception": false,
    "exception_traceback": null,
    "exception_message": null
  },
  "result": {
    "observed_value": 1184,
    "element_count": 10000,
    "missing_count": null,
    "missing_percent": null
  },
  "success": true,
  "expectation_config": {
    "kwargs": {
      "column": "trip_distance",
      "min_value": 1000,
      "max_value": 6000,
      "result_format": "BASIC"
    },
    "meta": {},
    "expectation_type": "expect_column_unique_value_count_to_be_between"
  }
}
[2022-09-08T18:49:21.665+0500] {python.py:173} INFO - Done. Returned value was: None
[2022-09-08T18:49:21.669+0500] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=expectation_dag, task_id=Expect_unique, execution_date=20220908T134905, start_date=20220908T134921, end_date=20220908T134921
[2022-09-08T18:49:21.723+0500] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-08T18:49:21.745+0500] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
